tools/javadoc-prettifier/src/test/java/io/spine/tools/protodoc/JavadocFormatterTest.java

```diff
@@ -37,37 +39,46 @@
 import static java.lang.System.lineSeparator;
 import static java.nio.charset.StandardCharsets.UTF_8;
 import static java.util.Collections.singletonList;
-import static org.junit.Assert.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junitpioneer.jupiter.TempDirectory.TempDir;
 
-public class JavadocFormatterShould {
+@ExtendWith(TempDirectory.class)
+@DisplayName("JavadocFormatter should")
+class JavadocFormatterTest {
 
     private static final char BACKTICK = '`';
     private static final String TEXT = "plain text";
     private static final String TEXT_IN_CODE_TAG = wrapWithCodeTag(TEXT);
     private static final String TEXT_IN_BACKTICKS = BACKTICK + TEXT + BACKTICK;
 
-    @Rule
-    public TemporaryFolder folder = new TemporaryFolder();
-
+    private File folder;
     private final FormattingAction formatting = new BacktickFormatting();
     private final List<FormattingAction> actions = singletonList(formatting);
     private final JavadocFormatter backtickFormatter = new JavadocFormatter(actions);
 
+    @BeforeEach
+    void setUp(@TempDir Path tempDirPath) {
+        folder = tempDirPath.toFile();
+    }
+
     @Test
-    public void ignore_files_except_java() throws IOException {
+    @DisplayName("ignore files expect java")
+    void ignore_files_except_java() throws IOException {
```

Comment:
Please find a way to fix the naming quickly, or create an issue.

Reply:
See #234.